contracts/constants.sol

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity 0.8.13;
+
+/// @dev The approximate number of blocks per year that is assumed by the interest rate model
+uint256 constant BLOCKS_PER_YEAR = 10_512_000;
```

Comment:
This shouldn't be a constant anymore. It should be immutable

Reply:
Removed here [7f93845](https://github.com/VenusProtocol/solidity-utilities/pull/4/commits/7f938459cad0743da83718dcb48f4678bed52c77)